core/builtins/native/kotlin/Library.kt

```diff
@@ -16,13 +16,31 @@
 
 package kotlin
 
+/**
+ * Returns true if the receiver and the [other] object are the same object instance, or if they
+ * are both null.
+ */
 public fun Any?.identityEquals(other: Any?): Boolean // = this === other
 
+/**
+ * Returns true if the receiver and the [other] object are "equal" to each other, or if they are
```

Comment:
Link to Any.equals() on "equal" would be good, to refer to explanation of equality.